### PR TITLE
Qmaps 445

### PR DESCRIPTION
**Pull request by Julien-laville**
first pass with new panel positionning for design v4

### CODE REVIEW

**Comment by nlehuby** [comment]
We will need a scrollbar on the POI panel
We also need a V-shaped mark on the opening hours block (so we can see it is collapsible)

capture d ecran - 2018-09-26 a 14 53 19

**Comment by nlehuby** [comment]
And where is the "Qwant Maps uses OSM" info block ? :cry:

**Comment by nlehuby** [comment]
The mobile view of the panel is quite ugly:
capture d ecran - 2018-09-26 a 15 06 25

File: src/scss/includes/panels/poi_panel.scss [hunk 1]
```diff
@@ -1,27 +1,22 @@
 $BLOCK_INDENT: 60px;
 
 .poi_panel {
```
[the remark this answers is not in the document]
Comment (Julien-laville) [reply src/scss/includes/panels/poi_panel.scss]: check

File: src/scss/includes/panels/service_panel.scss [hunk 2]
```diff
@@ -0,0 +1,39 @@
+.service_panel {
+  position: relative;
+  width: $panel_width;
+  height: 100px;
+  border-right: 1px solid #f3f5f7;
+  margin-top: 14px;
+  background: url('/images/popup_top.svg') white repeat-x;
```
[the remark this answers is not in the document]
Comment (Julien-laville) [reply src/scss/includes/panels/service_panel.scss]: check

File: src/scss/includes/top_bar.scss [hunk 3]
```diff
@@ -1,53 +1,18 @@
...
-  height: 95px;
   background: #fff;
-  border-bottom: 1px solid #f2f2f2;
-  top: 0;
-  right: 0;
-}
-
-.osm_contribute {
+  background-image: url('../images/qwant-logo.svg');
+  background-position: 15px 10px;
```
[the remark this answers is not in the document]
Comment (Julien-laville) [reply src/scss/includes/top_bar.scss]: check (merci)

File: src/scss/includes/app.scss [hunk 4]
```diff
@@ -2,6 +2,42 @@ BODY {
   background: url("../images/grid.png");
 }
 
+.side_panel__container {
+  max-height: calc(100vh - 122px);
```
[the remark this answers is not in the document]
Comment (Julien-laville) [reply src/scss/includes/app.scss]: - poi done
- fav

File: package-lock.json [hunk 5]
```diff
@@ -168,9 +168,9 @@
       }
     },
     "@mapbox/whoots-js": {
-      "version": "3.0.0",
```
[the remark this answers is not in the document]
Comment (Julien-laville) [reply package-lock.json]: fixed

**Comment by Julien-laville** [comment]
> And where is the "Qwant Maps uses OSM" info block ?

with the burger menu, (wait for new design)

**Comment by nlehuby** [comment]
missing image for the services panel + misalignment on mobile view in the autocomplete bar:
screenshot_2018-09-27 courtepaille

**Comment by nlehuby** [comment]
some panel dimensioning magic is still needed + maybe put the panels above the map control 
screenshot_2018-09-27 courtepaille 1

File: package-lock.json (hunk 5 above)
[the remark this answers is not in the document]
Comment (amatissart) [reply package-lock.json]: There are still vulnerabilities here :cry: 
`found 9 vulnerabilities (2 low, 5 moderate, 2 high)`

File: src/scss/includes/panels/favorite_panel.scss [hunk 6]
```diff
@@ -148,41 +125,29 @@
...
-}
-
-.favorite_panel__swipe_element {
-  position: relative;
-  overflow: hidden;
-  width: $panel_width;
-  height: 70px;
-  padding: 0 8px 0 4px;
-  transition: height .2s;
-  cursor: pointer;
```
Comment (amatissart) [inline src/scss/includes/panels/favorite_panel.scss]: We lost the "pointer" on favorite elements :cry:

Comment (Julien-laville) [reply src/scss/includes/panels/favorite_panel.scss]: fixed on 431

File: src/scss/includes/panels/favorite_panel.scss [hunk 7]
```diff
@@ -63,6 +38,8 @@
 
 .favorite_panel__item {
   font-size: 18px;
+  width: 385px;
```
[the remark this answers is not in the document]
Comment (Julien-laville) [reply src/scss/includes/panels/favorite_panel.scss]: will be fixed on 431

[the remark this answers is not in the document]
Comment (Julien-laville) [reply src/scss/includes/panels/favorite_panel.scss]: > some panel dimensioning magic is still needed + maybe put the panels above the map controls
> screenshot_2018-09-27 courtepaille 1

Mais qui a un écran aussi étroit ? 😂😂😂

**Pull request merged**